src/orquestra/sdk/_base/cli/_dorq/_arg_resolvers.py

```diff
@@ -207,9 +207,10 @@ def resolve_project_id(
         if optional:
             projects.append(None)
             labels.append("All")
-        prompt_choices = [(label, ws) for label, ws in zip(labels, projects)]
+        prompt_choices = [(label, project) for label, project in zip(labels, projects)]
 
-        return self._prompter.choice(prompt_choices, message="Projects")
+        selected_id = self._prompter.choice(prompt_choices, message="Projects")
+        return selected_id.project_id if selected_id else selected_id
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion (non-blocking):** ‏I think being a bit more verbose pays off
```suggestion
        if selected_id is not None:
            return selected_id.project_id
        return None
```